### PR TITLE
Move all qa_* jobs to a file owned by devx loops and e2e testing teams

### PR DESCRIPTION
**Pull request by KevinFairise2**
### What does this PR do?

It moves job that deploy packages used in e2e tests to a separate file so we can attribute the correct ownership and devxloops can be aware of changes on these files

### Motivation

Avoid modification on deploy jobs that we are not aware

### CODE REVIEW

File: .gitlab/deploy_packages/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Job that deploys agent packages on QA environment, to be used by e2e tests
```
Comment (L3n41c) [inline .gitlab/deploy_packages/e2e.yml:1]:
```suggestion
# Jobs that deploy agent packages on QA environment, to be used by e2e tests
```

File: .gitlab/dev_container_deploy/e2e.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# Job that deploys agent images on agent-qa ECR, to be used by e2e tests
```
Comment (L3n41c) [inline .gitlab/dev_container_deploy/e2e.yml:1]:
```suggestion
# Jobs that deploy agent images on agent-qa ECR, to be used by e2e tests

```

**Review by L3n41c — APPROVED** [review]

File: .gitlab/dev_container_deploy/e2e.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    # - !reference [.on_container_or_e2e_changes]
+    - !reference [.manual]
+  needs:
+    - docker_build_ot_agent7
+    - docker_build_ot_agent7_arm64
+  variables:
+    IMG_REGISTRIES: agent-qa
+    IMG_SOURCES: ${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot-beta-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot-beta-arm64
+    IMG_DESTINATIONS: agent:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot-beta
+qa_dca:
```
Comment (f4usto) [inline .gitlab/dev_container_deploy/e2e.yml:36]:
```suggestion
    IMG_DESTINATIONS: agent:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot-beta

qa_dca:
```

**Review by f4usto — APPROVED** [review]
nitpick

**Review by amenasria — APPROVED** [review]
LGTM for Agent DevX Infra owned files !

**Pull request merged**